partcad/src/partcad/part_factory_sweep.py

```
@@ -0,0 +1,178 @@
+                exp = TopExp_Explorer(faces, TopAbs_FACE)
+                while exp.More():
+                    face = exp.Current()
+                    maker = BRepOffsetAPI_MakePipe(self.axis_approx, face)
+                    maker.Build()
+                    shape = maker.Shape()
+                    builder.Add(compound, shape)
+                    exp.Next()
+
+                shape = compound
+
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)